src/waypoint_cmd.cpp

```diff
@@ -39,8 +39,13 @@
 void Waypoint::UpdateVirtCoord()
 {
 	Point pt = RemapCoords2(TileX(this->xy) * TILE_SIZE, TileY(this->xy) * TILE_SIZE);
```

Comment:
I think you forgot to change this line.

Reply:
I'm going to do the opposite, change the regular station one back. The only reason the RemadCoords() not-2 change was made originally was to solve a crash issue during loading some savegames from specific versions. I think this new approach will avoid those issues in a better way.

Reply:
That's the old, more correct method. Changing the one for regular stations back instead.